.gitignore

```diff
@@ -64,3 +64,7 @@ doc/Makefile
 
```

Comment:
@dianacretu this commit looks good and it is very useful. Anyhow, please add it in a separate PR because it is not related with 'Add qemu check for i.MX8 platform'.

We can also merge it earlier! :)